.github/workflows/lint.yml

```diff
@@ -11,22 +11,30 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  DDS_ROOT: ${{github.workspace}}
+  DDS_ROOT: ${{github.workspace}}/OpenDDS
   DOC_ROOT: ${{github.workspace}}/ACE_TAO
   ACE_ROOT: ${{github.workspace}}/ACE_TAO/ACE
 
 jobs:
-  build:
+  lint:
     runs-on: ubuntu-latest
     steps:
     - name: checkout OpenDDS
       uses: actions/checkout@v2
+      with:
+        path: ${{ env.DDS_ROOT }}
     - name: checkout ACE_TAO
       uses: actions/checkout@v2
       with:
         repository: DOCGroup/ACE_TAO
         ref: ace6tao2
         path: ${{ env.DOC_ROOT }}
+    - name: Install Perl Dependencies
+      uses: shogo82148/actions-setup-perl@v1
+      with:
+        install-modules: |
+          YAML
     - name: Run lint.pl
-      run: |
-        perl $DDS_ROOT/tools/scripts/lint.pl --color --ace
+      run: cd $DDS_ROOT ; perl tools/scripts/lint.pl --color --ace
+    - name: Run lint_build_and_test.pl
+      run: perl $DDS_ROOT/.github/workflows/lint_build_and_test.pl
```

Comment:
Should this script live in `tools/scripts`?  Would a developer run it locally?

Reply:
I see `tools/scripts` as mostly scripts that apply to multiple places or all of the repo, so I'm hesitant to move it away from the only file it's checking. A developer might want to run it, but they'd have to remember and it might be easier to remember if the script is in the same directory as the file they're editing.